shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -1634,7 +1637,7 @@ object GlutenConfig {
       .createWithDefault(true)
 
   val DYNAMIC_OFFHEAP_SIZING_ENABLED =
-    buildConf("spark.gluten.memory.dynamic.offHeap.sizing.enabled")
+    buildStaticConf("spark.gluten.memory.dynamic.offHeap.sizing.enabled")
```

Comment:
is this a behavior change by changing this to static config?

Reply:
Not really, it's just an improvement for configuration definitions.